language/move-lang/src/parser/syntax.rs

```diff
@@ -1747,7 +1787,10 @@ fn parse_constant_decl(
 //          "}"
 //
 // Note that "address" is not a token.
-fn parse_address_block(tokens: &mut Lexer) -> Result<(Loc, Address, Vec<ModuleDefinition>), Error> {
+fn parse_address_block(
```

Comment:
The grammar comment for `AddressBlock` needs to be updated, including the optional `<AddressBytes>`